modules/oauth/src/main/java/org/mule/module/oauth2/internal/authorizationcode/AuthorizationRequestHandler.java

```diff
@@ -89,7 +89,6 @@ public void init() throws MuleException
             final HttpListenerBuilder httpListenerBuilder = new HttpListenerBuilder(muleContext);
             final String flowName = "authorization-request-handler-" + localAuthorizationUrl;
             final Flow flow = DynamicFlowFactory.createDynamicFlow(muleContext, flowName, createLocalAuthorizationUrlListener());
-            muleContext.getRegistry().registerObject(flow.getName(), flow);
```

Comment:
because the flow is automatically registered by DynamicFlowFactory.createDynamicFlow.